### ISSUE TITLE
"One or more of your installed extensions will not be compatible with a feature Visual Studio update" warning in VS2019

### ISSUE THREAD

**Issue by zgabi** [deadlydog/VS.DiffAllFiles#30]
VS 2019 shows the following warning for DiffAllFiles extension:
One or more of your installed extensions will not be compatible with a feature Visual Studio update

Performance Manager shows that this extension is using a deprecated API.

It's not a critical issue, but sooner or later it should be fixed.

**Comment by deadlydog** [comment]
Yeah, this is a known issue. Thanks though.

I originally had it working using the new `AsyncPackage` as recommended, but it introduced issue #27 so I rolled the change back. I inquired how I should go about querying the "older" synchronous services in an async manner on startup [here](https://github.com/MicrosoftDocs/visualstudio-docs/issues/3058), but haven't gotten a great answer yet, and I think I'll need to inquire about it somewhere else.
